### PR TITLE
Paranoid for contact us and fix button name to 'Send message'

### PR DESCRIPTION
**Pull request by met0ry**
- [x] merged with the latest master branch
- [x] resolved all the conflicts, PR could be merged
- [x] new code has new specs
- [x] for new models - new factories were added too
- [x] unit specs are passed
- [x] code is well-formatted, Ruby, CSS, etc
- [x] screenshots of new pages, features are added
- [x] code passed internal team code review
- [x] no instance variables in partials
- [x] code is well formatted and has spaces!!!

PS: delete your branch in github after merge http://prntscr.com/q80mvq

### CODE REVIEW

**Comment by met0ry** [comment]
image
image

File: app/controllers/account/feedbacks_controller.rb [hunk 1]
```diff
@@ -1,17 +1,27 @@
...
-    @feedbacks = Feedback.by_creation_date
+    @feedbacks = Feedback.with_deleted.by_creation_date
   end
 
   def destroy
-    @feedback = Feedback.find(params[:id])
-    if @feedback.destroy
-      flash[:success] = 'Feedback successfully deleted.'
+    @feedback = Feedback.with_deleted.find(params[:id])
+    if @feedback.deleted_at == nil
```
Comment (romaslmd) [inline app/controllers/account/feedbacks_controller.rb]: `.nil?`

Comment (igorkasyanchuk) [reply app/controllers/account/feedbacks_controller.rb]: зміни на те що Роман каже

File: app/views/account/feedbacks/index.html.slim [hunk 2]
```diff
@@ -11,9 +11,13 @@ h4.card-title Feedbacks
       .information.d-sm-flex.d-none.align-items-center
         small.text-muted.mr-5
           = l(feedback.created_at, format: :long)
-        = link_to 'Delete', account_feedback_path(feedback), method: :delete
+        - if feedback.deleted_at == nil
```
Comment (igorkasyanchuk) [inline app/views/account/feedbacks/index.html.slim]: і тут

**Pull request merged**